README.md

```diff
@@ -510,9 +510,13 @@ To compress PNGs and SVGs in the current directory and its subdirectories, two c
 
 # Releasing
 
-1. Create a pull request. Get it reviewed and approved.
-1. Run `npm version patch`, `npm version minor`, or `npm version major` as appropriate.
-1. Merge the pull request.
+1. Create a pull request with prep for the new release. This should contain changes to release notes and anything else that needs to be done.
+1. Get it reviewed and approved, and merge the pull request.
+1. Run `npm version patch`, `npm version minor`, or `npm version major` as appropriate. This will:
```

Comment:
if you run `npm version patch` after the pr is merged, you'll need to create a new pr for version change. that is why the previous instructions required you to run `npm version` before merging the pull request.

Reply:
OH, because otherwise you can't push the changes into master?

It confused me because `npm version patch` is actually doing "the business", so it's sort of closing the barn doors after the horse has bolted.

Is the idea in the original instructions that you have an empty PR that is rubber stamped by someone, giving you the permission to merge the changes after `npm version xyz` has generated them?

Reply:
right. you can't push directly to master.

in practice, we end up making a pr just for the version bump, because people forget
but imo, the smoothest process is to just run npm version after sign-off but before merging the pr that is getting published.